packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1758,12 +1758,41 @@ You can define multiple IdP instances in a single Policy Action. This allows use
 | type                               | Provider type. Possible values: `OKTA`, `AgentlessDSSO`, `IWA`, `X509`, `SAML2`, `OIDC`, `APPLE`, `FACEBOOK`, `GOOGLE`, `LINKEDIN`, `MICROSOFT`           | String | Yes |
 | name <ApiLifecycle access="ie" />  | Provider `name` in Okta | String    | No       |
 
+#### Policy Action with Dynamic IdP routing
```

Comment:
just below this title, we need to add the EA flag - which is the following (without the spaces after the first bracket and after the slash). :)

< ApiLifecycle access="ea" / >